src/flypad-backend/src/Aircraft/AircraftProcedures.h

```diff
@@ -237,29 +238,47 @@ class AircraftProcedures {
 
   template<std::size_t N1, std::size_t N2>
   static constexpr void doInsertProcedures(std::array<const ProcedureStep*, N1>& dest, const std::array<ProcedureStep, N2>& src, const std::size_t offset) {
-    std::transform(begin(src), end(src), begin(dest) + offset * N2, [](const auto& procedure) {
+    std::transform(begin(src), end(src), begin(dest) + offset, [](const auto& procedure) {
       return &procedure;
     });
   }
 
   template<class Tuple, std::size_t S, std::size_t... I>
   static constexpr void doInsertProcedures(std::array<const ProcedureStep*, S>& dest, const Tuple& procedures, std::index_sequence<I...>) {
-    (doInsertProcedures(dest, std::get<I>(procedures), I), ...);
+    (doInsertProcedures(dest, std::get<I>(procedures), (I == 0 ? 0 : I - 1) * std::get<(I == 0 ? 0 : I - 1)>(procedures).size()), ...);
```

Comment:
Also this overload is never called and I think has not the correct semantics so I suggest to remove it

Reply:
Oh whoops that's still my little sandbox environment